src/uct/ib/rdmacm/rdmacm_def.h

```diff
@@ -34,6 +35,11 @@ typedef struct uct_rdmacm_ctx {
     ucs_list_link_t    list;    /* for list of used cm_ids */
 } uct_rdmacm_ctx_t;
 
+static inline size_t uct_rdmacm_cm_get_max_conn_priv()
```

Comment:
can remove `inline`, this is not a fast path and most likely will be inlined by compiler automatically.